pkg/services/hcloud/server/server.go

```
@@ -465,8 +465,30 @@ func (s *Service) createServer(ctx context.Context) (*hcloud.Server, error) {
 				s.scope.Name(),
 				err,
 			)
-			return nil, errServerCreateNotPossible
+
+			err = errors.Join(errServerCreateNotPossible, err)
```

ah now I saw what this error is used for. I think that in both cases that we have there, we should return this error as well to not immediately reconcile. WDYT?

---

seems logical ... then it would reque after 5 min...

---

so it this correctly implemented right now?

---

I guess it is. then we can resolve it

---

Yes